valhalla/mjolnir/landmark_database_builder.h

```diff
@@ -0,0 +1,81 @@
+#pragma once
+
+#include <boost/property_tree/ptree.hpp>
+#include <cstdint>
+#include <iostream>
+#include <list>
+#include <string>
+#include <vector>
+
+#include "baldr/graphconstants.h"
+#include "filesystem.h"
+#include "mjolnir/util.h"
+
+#include <boost/geometry.hpp>
+#include <boost/geometry/geometries/geometries.hpp>
+#include <boost/geometry/geometries/register/point.hpp>
+#include <boost/property_tree/ptree.hpp>
+
+#include <geos_c.h>
```

Comment:
all of these except for ptree are probably not even used. we should avoid including things that we do not use, its not free. google the "include what you use" to learn more about why its a bad thing to take more than you need

Reply:
Thanks for pointing it out. They were copy paste from other files... I have removed the unused imports. It turns out we don't even need p_tree for now but I just comment it in case we need it in the next PR.